src/state/GlobalState.tsx

```diff
@@ -249,33 +251,42 @@ export class GlobalState extends PureComponent<Props> {
   }
 
   async componentDidMount() {
-    const { i18n } = this.props
+    const { i18n, t } = this.props
+    const { gameUpdates, libraryStatus } = this.state
+
+    // Deals launching from protocol. Also checks if the game is already running
+    ipcRenderer.once('launchGame', async (e, appName) => {
```

Comment:
Why not use `ipcRenderer.on`?

Reply:
because once will guarantee that the function won't launch the game twice. I'm also checking if the game have some status, to avoid other errors.